packages/core/src/app/customer/customWalletButton/AmazonPayV2Button.tsx

```diff
@@ -26,8 +26,10 @@ const beautifyAmazonButton = (): void => {
 
     if (container && amazonButton) {
         amazonButton.style.height = '36px';
+        container.style.width = '100%';
+        
         return;
     }
 
-    setTimeout(beautifyAmazonButton, 10);
+    setTimeout(beautifyAmazonButton, 200);
```

Comment:
Why are we changing the timeout?

Reply:
I changed timeout to reduce beautifyAmazonButton function calls. I set it to 200 as it still looks ok then.